i18n/pl/cosmic_settings.ftl

```diff
@@ -108,11 +201,50 @@ style = Styl
     .slightly-round = Lekko zaokrąglony
     .square = Kwadratowy
 
-# interface density left out for now
+interface-density = Zagęszczenie Interfejsu
+    .comfortable = Wygodne
+    .compact = Zwarte
+    .spacious = Swobodne
+
 window-management-appearance = Zarządzanie Oknami
     .active-hint = Rozmiar wyróżnienia aktywnego okna
     .gaps = Przerwa między ramkami okien w trybie kafelków
 
+### Experimental
+
+experimental-settings = Ustawienia Eksperymentalne
+icons-and-toolkit = Motywy biblioteki i ikon
+interface-font = Czcionka systemowa
+monospace-font = Czcionka o stałej szerokości
+
+### Appearance: Font
+
+font-config = Konfiguracja Czcionek
+interface-font = Czcionka Systemowa
+monospace-font = Czcionka o Stałej Szerokości
+font-family = Rodzina
+
+font-weight = Grubość
```

Comment:
Not used

Reply:
### Experimental

experimental-settings = Ustawienia Eksperymentalne
icons-and-toolkit = Motywy biblioteki i ikon
interface-font = Czcionka systemowa
monospace-font = Czcionka o stałej szerokości

Isn't that bit used in EN translation 206-211?